### ISSUE TITLE
TaglibSharp 2.2.0 has been released

### ISSUE THREAD

**Issue by decriptor** [lidarr/taglib-sharp#10]
If you all are willing it'd be nice to close this repo and just work on the upstream version. I believe I've merged all of the changes at this point and published a new nuget package.

https://github.com/mono/taglib-sharp/releases/tag/TaglibSharp-2.2.0

**Comment by Qstick** [comment]
@decriptor I don’t think we mind blitzing the package, but the main reason we started was due to long periods of unresponsiveness on the main repo. This was causing issue with our user base and implementation that we needed to solve. Do you feel confident that when issues come up that they can/will be addressed quickly?

**Comment by decriptor** [comment]
@Qstick Funny enough you caught me on vacation. I think now that things are a bit more cleaned up and simplified I'm hoping to have more regular responsive release. If you need a PR merged and a release done quickly I think I'm better able to do that now.

**Comment by ta264** [comment]
@decriptor If I PR https://github.com/lidarr/taglib-sharp/pull/12 upstream any chance of a merge + release in the next few days?  Turns out lidarr is corrupting OPUS files on tag write 😬

**Comment by decriptor** [comment]
@ta264 Let me see if I can knock something out this weekend.

**Comment by ta264** [comment]
Thanks! You can hit me up on the lidarr discord if you want to discuss anything

**Comment by decriptor** [comment]
@ta264 Would you be ok creating a PR against mono/taglib-sharp and create a test? If not, I'll create a PR and try to put together a unit test. Let me know.

**Comment by ta264** [comment]
I was testing locally by calling out to opusinfo to verify integrity. Would that be acceptable or do you want it to be totally self contained?

**Comment by decriptor** [comment]
@ta264 It is so that CI can run the test and make sure we don't regress when new PRs come in. Essentially making sure this bug isn't regressed again.
